driver/99-stlink-plugdev.rules

```diff
@@ -0,0 +1,30 @@
+# ST-Link Debug Probes
+#
+# Copy this file to /etc/udev/rules.d/99-stlink-plugdev.rules
+# and run sudo udevadm control -R
+#
+# ST-Link v1
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="3744", TAG+=uaccess, MODE="0664", GROUP="plugdev"
+#
+# ST-Link v2
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="3748", TAG+=uaccess, MODE="0664", GROUP="plugdev"
+#
+# ST-Link v2.1
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="374a", MODE="0664", TAG+=uaccess, GROUP="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="374b", MODE="0664", TAG+=uaccess, GROUP="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="3752", MODE="0664", TAG+=uaccess, GROUP="plugdev"
+#
+# STLink V3SET in Dual CDC mode
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="3752", TAG+=uaccess, MODE="0664", GROUP="plugdev"
+#
+# STLink V3SET in Dual CDC mode
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="3753", TAG+=uaccess, MODE="0664", GROUP="plugdev"
+#   
+# STLink V3SET 
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="374d", TAG+=uaccess, MODE="0664", GROUP="plugdev"
+#
+# STLink V3SET 
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="374e", TAG+=uaccess, MODE="0664", GROUP="plugdev"
+#   
+# STLink V3SET in normal mode
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="374f", TAG+=uaccess, MODE="0664", GROUP='plugdev"
```

Comment:
Could you add an empty line at the end please so Git isn't mad about the lack of newline on the final line (also, it's possible some udev versions will be upset if it's not present)